jobs/config.json

```diff
@@ -9300,6 +9300,15 @@
       "sig-network"
     ]
   },
+  "pull-kubernetes-multicluster-ingress-test": {
+    "args": [
+      "true"
```

Comment:
do you actually want to run something useful? xD

Reply:
Added "make test" here. It should start working once https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/pull/62 merges.